sources/INSPECT

```diff
@@ -1303,13 +1360,14 @@ Copyright (c) 1982-1987, 1990-1991, 1993, 1995, 1999, 2018, 2021 by Venue & Xero
     (for PROP in (PROPNAMES ATM) when (NOT (FMEMB PROP SYSPROPS)) collect PROP])
 
 (INSPECT/LISTP
-  [LAMBDA (LST WHERE)                                    (* ; "Edited  2-Feb-87 17:05 by jop")
+  [LAMBDA (LST WHERE TAG)                                    (* ; "Edited 12-Sep-2022 20:49 by rmk")
```

Comment:
is there an example of how TAG is used?
Should there be some 'release note' that explains new capabilities so others can explore?

Reply:
> is there an example of how TAG is used?
> Should there be some 'release note' that explains new capabilities so others can explore?

We should figure out a release-note strategy, to track extensions and revisions in general.  A single document, a collection of notes in a well-defined place?

As for tag, there are 2 ways:  give a tag as the optional argument in the call to INSPECT, or use the title button-function to add a tag to an existing inspect window.

A separate issue:  INSPECT current returns NIL, so that there is no way of manipulating the inspect window after the fact (e.g. to have a program that move it around in a systematically helpful way).  Perhaps INSPECT should return its window?